docs/testing/cli_testing_frameworks.md

```diff
@@ -4,22 +4,22 @@ title: Stacks Testing Frameworks
 sidebar_label: Stacks Testing Frameworks
 ---
 
-Building a testing frameworks from scratch requires a lot of boilerplate code. It is also hard to know how to get started without examples.
+Building a testing framework from scratch requires a lot of boilerplate code. It is also hard to know how to get started without examples.
 
 As part of Amido Stacks projects, we have developed a range of Javascript Front End focused, opinionated, boilerplate testing frameworks that can be bootstrapped and tailored to your current project with just one command.
 
 ## Bootstrap your framework
 
-The best way to get started, is bootstrap the framework using the scaffolding-cli from Amido Stacks. By using it in interactive mode, you can simply answer 5 questions to get a bespoke framework, ready to go with an example test.
+The best way to get started is to bootstrap the framework using the scaffolding-cli from Amido Stacks. By using it in the interactive mode, you can simply answer 5 questions to get a bespoke framework, ready to go with an example test.
```

Comment:
The best way to get started is to bootstrap the framework using the scaffolding-cli from Amido Stacks. By using it in interactive mode, you can simply answer 5 questions to get a bespoke framework ready to go with an example test.